services/schedview/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for schedview.
+# This is a YAML-formatted file.
```

Comment:
To look more uniform with the other files, remove the top 3 lines here. To be clear the file should start at `# -- Number ...`